<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayScence</class>
 <widget class="QWidget" name="PlayScence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img_label2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img_label3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="info_label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
